scenes/game/local_game.gd

```diff
@@ -814,6 +835,20 @@ class Player:
 		gauge = []
 		return events
 
+	func swap_deck_and_sealed():
+		var events = []
+		var current_sealed_ids = sealed.map(func(card) : return card.id)
+		var current_deck_ids = deck.map(func(card) : return card.id)
+		for card_id in current_deck_ids:
+			var seal_effect = { "effect_type": "seal_card_INTERNAL", "seal_card_id": card_id, "source": "deck", "silent": true }
```

Comment:
check out my arakune branch for what I did by making a function do_seal_effect.
Though you can just leave that to me to merge if you don't feel like taking it.
Having this effect copied everywhere is messy.